lib/editor/components/timetable/TimetableHelpModal.js

```diff
@@ -0,0 +1,69 @@
+import React, {Component, PropTypes} from 'react'
+import {Modal, Button} from 'react-bootstrap'
+
+import {getComponentMessages, getMessage} from '../../../common/util/config'
+
+const SHORTCUTS = {
+  offset: ['o', 'SHIFT:+:o', 'i', 'SHIFT:+:i', '-', 'SHIFT:+:-', '+', 'SHIFT:+:+'],
+  navigate: ['k:/:j', '←:/:→', 'x', 'a', 'd'],
+  modify: ['#', 'n', 'c', 'SHIFT:+:\'', 'SHIFT:+:;']
+}
+
+export default class TimetableHelpModal extends Component {
+  static propTypes = {
+    show: PropTypes.bool
+  }
+
+  state = {
+    showModal: this.props.show
+  }
+
+  messages = getComponentMessages('TimetableHelpModal')
```

Comment:
This should live outside the class since it won't change based on the attributes of each instance..